src/extension.ts

```diff
@@ -12,6 +12,8 @@ import { ModuleTwinCodeLensProvider } from "./providers/moduleTwinCodeLensProvid
 import { TelemetryClient } from "./telemetryClient";
 
 export function activate(context: vscode.ExtensionContext) {
+    TelemetryClient.extensionContext = context;
```

Comment:
Better to use a initialize function to wrap it and make extensionContext as a private property, refer to https://github.com/Microsoft/vscode-azure-iot-toolkit/blob/96d1367e361c99fd0f18d3ea20f016973992b74f/src/deviceTree.ts#L15:5.

And put this line just before `Constants.initialize(context);`  and after `TelemetryClient.sendEvent("extensionActivated");`